docs/chapters/developer_guide.tex

```diff
@@ -0,0 +1,176 @@
+
+\begin{Remark}
+	\item This configuration can not be used to build a standalone executable with nuitka tooling. It is for development purposes only.
+\end{Remark}
+
+\section{Configuration}
+The tool can be configured with a batch script which will configure using installed tools or ask the user to download and install the tools which can be used for them. The batch file is located in the sub folder \textbf{BuildScripts} and is called \textbf{DevelopDfastmi.bat}.
+
+The batch file will check for installed tooling for anaconda/miniconda. If not installed it will ask the user to install the anaconda/miniconda tooling for you. 
+\begin{Remark}
+	\item \textbf{PLEASE NOTE:} you need to close the command line and re-open / restart the batch script as it needs to initialize the conda tooling for the command line and powershell (used in VSCode).
+\end{Remark}
+
+The batch file will check for installed tooling for VSCode. If not installed it will ask the user to install the VSCode tooling for you. 
+
+\begin{Remark}
+	\item \textbf{PLEASE NOTE:} you need to close the command line and re-open / restart the batch script as it needs to initialize the VS Code tooling for the command line.
+\end{Remark}
+
+
+\section{Configuration files}
+As stated earlier we use conda to create a clean python environment to develop the \dfastmi application in. We also use poetry to setup the needed packages for the development environment. Poetry uses a toml\footnote{TOML : Tom's Obvious Minimal Language. A config file format file} file for the configuration (\textbf{poetry.toml}) and a lock file (\textbf{poetry.lock}) which i have no clue why it is there. Also we use a \textbf{setup.py} file used by the setup tools package to setup the application for development. And last but not least a python tools / interpreter configuration \textbf{pyproject.toml}. This file configures the way python packages should behave.
+
+\subsection{Conda environment}
+We setup a conda environment on the client pc so we have the python interpreter we expect to be used by developers.
+\begin{table}[]
+	\caption{Default packages for conda environment with python interpreter v3.9.13.}
+	\begin{tabular}{|l|l|}
+		\hline
+		\textbf{Package}    & \textbf{Remark} \\ \hline
+		ca-certificates     &                 \\ \hline
+		openssl             &                 \\ \hline
+		pip                 &                 \\ \hline
+		python              &                 \\ \hline
+		setuptools          &                 \\ \hline
+		sqlite              &                 \\ \hline
+		tzdata              &                 \\ \hline
+		vc                  &                 \\ \hline
+		vs2015\_runtime     &                 \\ \hline
+		wheel               &                 \\ \hline
+	\end{tabular}
+\end{table}
+
+\subsubsection{Conda / Python packages needed for \dfastmi}
```

Comment:
Where did you get these packages from?

Aren't the needed packages stated in the PyProject.toml and poetry.lock file?

Reply:
Is there a specific reason to add these packages in a list in the user guide?